gcn/circulars.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/circulars.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "GCN Circular",
+  "description": "GCN Circulars are rapid astronomical bulletins submitted by and distributed to community members worldwide. They are used to share discoveries, observations, quantitative near-term predictions, requests for follow-up observations, or future observing plans related to high-energy, multi-messenger, and variable or transient astrophysical events. See https://gcn.nasa.gov/circulars",
+  "properties": {
+    "eventId": {
```

Comment:
How about using core schema: https://gcn.nasa.gov/docs/schema/v4.0.0/gcn/notices/core/Event.schema.json ?
It can be same property for both Circulars and Notices, helpful for correlation.

Reply:
We could, but we would need to migrate the whole GCN Circulars database if we changed field names.

Reply:
Ohh, may be one more translation layer can help later.
@jracusin what is your opinion keeping the name as available in Unified Schema or it's fine?

Reply:
I don't think we need a translation layer. I agree with you, @Vidushi-GitHub, that this schema is not very consistent with GCN Notices. We should discuss if and how we should achieve this transition.